### PR TITLE
Implemented virga fusion and shape change on clivis.

### PR DESCRIPTION
**Pull request by henryso**
Implementation for #1558

### CODE REVIEW

**Comment by henryso** [comment]
Don't know how I closed this.  Maybe I pressed some hot key in the window.

File: src/dump/dump.c [hunk 1]
```diff
@@ -424,9 +424,13 @@ void dump_write_score(FILE *f, gregorio_score *score)
                                         gregorio_texverb(note->texverb));
                             }
                             if (note->choral_sign) {
-                                fprintf(f, "         Choral Sign            \"%s\"\n",
+                                fprintf(f, "         choral sign            \"%s\"\n",
```
Comment (rpspringuel) [inline src/dump/dump.c:427]: Why is this changing?  It doesn't seem related to what's being implemented here.

**Review by rpspringuel — COMMENTED** [review]
The tests of the new shapes look good.  My only issue is what seems to be a superfluous change to the capitalization of the dump output for the choral sign.  Am I missing something as to why that needs to change?

File: src/dump/dump.c (hunk 1 above)
Comment (henryso) [reply src/dump/dump.c:427]: This wasn't there before.  I added it (and then de-capitalized it) because I expected a change to affect that.

Comment (rpspringuel) [reply src/dump/dump.c:427]: I read the changes as showing the "Choral Sign" was there before and the only change is the change in capitalization.  What am I missing?

Comment (henryso) [reply src/dump/dump.c:427]: I must be conflating this with something.  So yes, I only changed the capitalization to be in line with the other text.

Comment (rpspringuel) [reply src/dump/dump.c:427]: Ah, I hadn't noticed the capitalization inconsistency.  I have no further problems with this change.
